src/ast/mod.rs

```diff
@@ -192,6 +215,12 @@ pub enum Expr {
     Identifier(Ident),
     /// Multi-part identifier, e.g. `table_alias.column` or `schema.table.col`
     CompoundIdentifier(Vec<Ident>),
+    /// JsonIdentifier eg: data->'tags'
+    JsonIdentifier {
```

Comment:
Since this is more like a map access, what do you think about calling this `JsonAccess` to be consistent with `MapAccess` (for `foo[4]` type expressions)?

Reply:
done